checks.d/mongo.py

```diff
@@ -875,3 +887,49 @@ def total_seconds(td):
 
         else:
             self.log.debug('"local" database not in dbnames. Not collecting ReplicationInfo metrics')
+
+        # get collection level stats
+        try:
+            # Ensure that you're on the right db
+            db = cli[db_name]
+            # grab the collections from the database
+            coll_names = db.collection_names()
+            collection_metrics = []
+            for m in self.COLLECTION_METRICS:
+                # grab all the collections
+                collection_metrics.append(m.split('.')[1:])
```

Comment:
Do we need to generate that list at each iteration ? It seems to be a constant to me.